src/libs/antares/study/area/links.cpp

```diff
@@ -591,23 +579,25 @@ bool AreaLinksLoadFromFolder(Study& study, AreaList* l, Area* area, const AnyStr
             case Data::tncIgnore:
             {
                 // Ignore transmission capacities
-                link.data.columnToZero(Data::fhlNTCDirect);
-                link.data.columnToZero(Data::fhlNTCIndirect);
+                link.directCapacities.zero();
+                link.indirectCapacities.zero();
                 break;
             }
             case Data::tncInfinite:
             {
                 // Copper plate mode
                 auto infinity = +std::numeric_limits<double>::infinity();
-                link.data.fillColumn(Data::fhlNTCDirect, +infinity);
-                link.data.fillColumn(Data::fhlNTCIndirect, +infinity);
+                link.directCapacities.fillColumn(0, +infinity);
+                link.indirectCapacities.fillColumn(0, +infinity);
                 break;
             }
             }
         }
 
         // memory swap
-        link.data.flush();
+        link.parameters.flush();
+        link.directCapacities.flush();
+        link.indirectCapacities.flush();
```

Comment:
`link.flush();` does that in 1 line

Reply:
you're right, I didn't call **flush()** on the link although I introduced this function in the link class